### ISSUE TITLE
SOGS passes the Lighthouse test

### ISSUE THREAD

**Issue by Oleg-Zhymolokhov** [Green-Software-Foundation/state-of-green-software#148]

**Comment by osamajandali** [comment]
@Oleg-Zhymolokhov I am getting 98%-100% on performance now. Give it a test, and let me know if you got lower numbers.

**Comment by Oleg-Zhymolokhov** [comment]
@osamajandali  Tested several times on different pages (Desktop)
<img width="544" alt="image" src="https://github.com/Green-Software-Foundation/state-of-green-software/assets/125550685/03ff5b14-c61f-4281-80b1-7ec6306ad32e">
<img width="546" alt="image" src="https://github.com/Green-Software-Foundation/state-of-green-software/assets/125550685/f21e42c5-fddc-480a-9349-4eaab4e59eb5">
<img width="549" alt="image" src="https://github.com/Green-Software-Foundation/state-of-green-software/assets/125550685/5cc0722e-028b-4b28-8967-d95162111364">

**Comment by osamajandali** [comment]
@Oleg-Zhymolokhov It may be caused due to a Chrome extension you have.
Can you give it a try in incognito mode?
Just keep in mind that the survey data page contains a lot of graphs, the score could be around 87.  I could spend time later on to improve this page but other than this page should score 100.

**Comment by zanete** [comment]
@Oleg-Zhymolokhov are you happy for us to close this issue?

**Comment by Oleg-Zhymolokhov** [comment]
@zanete yes, let's close it and I'll open a new one when/if we'll make changes to the website.
